src/PKSim.CLI.Core/Services/SimulationExporter.cs

```diff
@@ -59,90 +59,92 @@ IMoBiExportTask moBiExportTask
          _moBiExportTask = moBiExportTask;
       }
 
-      public async Task RunAndExport(Simulation simulation, string outputFolder, SimulationRunOptions simulationRunOptions, SimulationExportMode simulationExportMode, string projectName = null)
+      public async Task RunAndExport(Simulation simulation, string outputFolder, SimulationRunOptions simulationRunOptions, SimulationExportMode simulationExportMode, string projectName = null, string category = null)
       {
-         _logger.AddDebug($"Running simulation '{simulation.Name}'");
+         var logCategory = logCategoryFrom(projectName, category);
+         _logger.AddDebug($"Running simulation '{simulation.Name}'", logCategory);
          await _simulationRunner.RunSimulation(simulation, simulationRunOptions);
 
-         await Export(simulation, outputFolder, simulationExportMode, projectName);
+         await Export(simulation, outputFolder, simulationExportMode, projectName, logCategory);
       }
 
-      public Task Export(Simulation simulation, string outputFolder, SimulationExportMode simulationExportMode, string projectName = null)
+      public Task Export(Simulation simulation, string outputFolder, SimulationExportMode simulationExportMode, string projectName = null, string category = null)
       {
+         var logCategory = logCategoryFrom(projectName, category);
          var tasks = new List<Task>();
          var individualSimulation = simulation as IndividualSimulation;
          var populationSimulation = simulation as PopulationSimulation;
 
          if (simulationExportMode.HasFlag(SimulationExportMode.Csv) && individualSimulation != null)
-            tasks.Add(exporIndividualSimulationToCsvAsync(individualSimulation, outputFolder, projectName));
+            tasks.Add(exporIndividualSimulationToCsvAsync(individualSimulation, outputFolder, projectName, logCategory));
 
          if (simulationExportMode.HasFlag(SimulationExportMode.Csv) && populationSimulation != null)
-            tasks.Add(exportPopulationSimulationToCsvAsync(populationSimulation, outputFolder, projectName));
+            tasks.Add(exportPopulationSimulationToCsvAsync(populationSimulation, outputFolder, projectName, logCategory));
 
          if (simulationExportMode.HasFlag(SimulationExportMode.Json) && individualSimulation != null)
-            tasks.Add(exportResultsToJsonAsync(individualSimulation, outputFolder, projectName));
+            tasks.Add(exportResultsToJsonAsync(individualSimulation, outputFolder, projectName, logCategory));
 
          if (simulationExportMode.HasFlag(SimulationExportMode.Xml))
-            tasks.Add(exportSimModelXmlAsync(simulation, outputFolder, projectName));
+            tasks.Add(exportSimModelXmlAsync(simulation, outputFolder, projectName, logCategory));
```

Comment:
This log cateogries will be very useful once we start to export in // . That way we know to which project the entry belongs